### PR TITLE
Fix the formatting of the XML documentation for the OnnxConverter

### PR DESCRIPTION
**Pull request by nandaleite**
Issue #1260

We are excited to review your PR.

So we can do the best job, please check:

- [ ] There's a descriptive title that will make sense to other developers some time from now. 
- [x] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [ ] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

File: src/Microsoft.ML.Legacy/Models/OnnxConverter.cs [hunk 1]
```diff
@@ -25,38 +25,42 @@ public sealed partial class OnnxConverter
...
+        /// <para>Learners that can be exported to ONNX</para>
+        /// <list type="number">
+        /// <item><description>FastTree</description></item>
+        /// <item><description>LightGBM</description></item>
+        /// <item><description>Logistic Regression</description></item>
+        /// </list>
         ///
-        /// See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/OnnxTests.cs"/>
-        /// for an example on how to train a model and then convert that model to ONNX.
+        /// <para>See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/OnnxTests.cs"/>OnnxTests.cs</a>
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: >/ [](start = 122, length = 1)

This one is extra, please remove it.

Comment (nandaleite) [reply src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: Thanks for pointing this out. I just fixed it.

File: src/Microsoft.ML.Legacy/Models/OnnxConverter.cs [hunk 2]
```diff
@@ -10,13 +10,13 @@ namespace Microsoft.ML.Legacy.Models
     public sealed partial class OnnxConverter
     {
         /// <summary>
-        /// <a href="https://onnx.ai/">ONNX</a> is an intermediate representation format
+        /// <para><a href="https://onnx.ai/">ONNX</a> is an intermediate representation format
         /// for machine learning models. It is used to make models portable such that you can
```
[the remark this answers is not in the document]
Comment (nandaleite) [reply src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: Hi @sfilipi. Just to clarify, are you suggesting to update the summary to:

```suggestion
///<summary>
/// <para><a href="https://onnx.ai/">ONNX</a> is an intermediate representation format
/// for machine learning models.</param>
///</summary>
```

Let me know. Thanks

[the remark this answers is not in the document]
Comment (sharwell) [reply src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: That is my understanding, yes. The rest of the text goes in `<remarks>`.

📝 Single paragraph summaries do not need explicit `<para>` elements, and most such cases would omit them.

[the remark this answers is not in the document]
Comment (nandaleite) [reply src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: Thank you, @sharwell

File: src/Microsoft.ML.Legacy/Models/OnnxConverter.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (nandaleite) [reply src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: Would `<see cref="OnnxTests"/>` be appropriate in this case?
I need more information on this. Thanks

File: src/Microsoft.ML.Legacy/Models/OnnxConverter.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (sfilipi) [reply src/Microsoft.ML.Legacy/Models/OnnxConverter.cs]: Looks good. Thanks!

---
In reply to: [226713686](https://github.com/dotnet/machinelearning/pull/1311#discussion_r226713686) [](ancestors = 226713686)

**Comment by shauheen** [comment]
Thanks @nandaleite this closes #1260

**Comment by nandaleite** [comment]
Thank you!

**Pull request merged**